examples/ConsoleApp/Program.cs

```diff
@@ -11,45 +10,37 @@
     .AddJsonFile("appsettings.json")
     .Build();
 
-//
-// Setup application services + feature management
-IServiceCollection services = new ServiceCollection();
+IFeatureDefinitionProvider featureDefinitionProvider = new ConfigurationFeatureDefinitionProvider(configuration);
```

Comment:
I think inlining this would be even better. 

```
var featureManager = new FeatureManager(new ConfigurationFeatureDefinitionProvider(configuration))
```

Shortens the sample even more.